images/linux/post-generation/environment-variables.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Replace $HOME with the default user's home directory for environmental variables related to the default user home directory
+
+default_user=$(cut -d: -f1 /etc/passwd | tail -1)
```

Comment:
We can request user home folder:
`homeDir=$(cut -d: -f6 /etc/passwd | tail -1)`

Reply:
Thanks!